tests/functional/test_config_groups.py

```diff
@@ -28,64 +28,86 @@
     return cluster_bundle.cluster_create(name=utils.random_string())
 
 
 @pytest.fixture()
-def provider(request, sdk_client_fs: ADCMClient) -> Provider:
-    bundle_path = request.param if hasattr(request, "param") else PROVIDER_BUNDLE_PATH
-    provider_bundle = sdk_client_fs.upload_from_fs(bundle_path)
+def provider(provider_bundle) -> Provider:
     return provider_bundle.provider_create(name=utils.random_string())
 
 
-class TestGroupsIntersection:
+@pytest.fixture()
+def create_two_hosts(provider) -> Tuple[Host, Host]:
+    with allure.step("Create host for config groups"):
+        test_host_1 = provider.host_create(fqdn=FIRST_HOST)
+    with allure.step("Create host for host candidate check"):
+        test_host_2 = provider.host_create(fqdn=SECOND_HOST)
+    return test_host_1, test_host_2
 
-    HOST_ERROR_MESSAGE = (
-        "host is not available for this object, or host already is a member of another group of this object"
-    )
-    FIRST_COMPONENT_NAME = "first"
-    SECOND_COMPONENT_NAME = "second"
-    FIRST_GROUP = "test_group"
-    SECOND_GROUP = "test_group_2"
-    FIRST_HOST = "test_host_1"
-    SECOND_HOST = "test_host_2"
 
-    @pytest.fixture()
-    def create_two_hosts(self, provider) -> Tuple[Host, Host]:
-        with allure.step("Create host for config groups"):
-            test_host_1 = provider.host_create(fqdn=self.FIRST_HOST)
-        with allure.step("Create host for host candidate check"):
-            test_host_2 = provider.host_create(fqdn=self.SECOND_HOST)
-        return test_host_1, test_host_2
-
-    @allure.step('Check error')
-    def assert_that_host_add_is_unavailable(self, service_group: GroupConfig, host: Host):
-        with allure.step(f"Check that error is '{GROUP_CONFIG_HOST_ERROR.code}'"):
-            with pytest.raises(ErrorMessage) as e:
-                service_group.host_add(host)
-            GROUP_CONFIG_HOST_ERROR.equal(e)
-        with allure.step(f"Check error message is '{self.HOST_ERROR_MESSAGE}'"):
-            assert (
-                self.HOST_ERROR_MESSAGE in e.value.error['desc']
-            ), f"Should be error message '{self.HOST_ERROR_MESSAGE}'"
-
-    @allure.step("Check that the only second host is present in candidates on second provider group")
-    def assert_host_candidate_equal_expected(self, group: HostList, expected_hosts_amount: int):
-        with allure.step(f"Check that {expected_hosts_amount} hosts are available in group"):
-            assert len(group) == expected_hosts_amount, f"{expected_hosts_amount} hosts should be available in group"
-        with allure.step(f"Check that host '{self.SECOND_HOST}' is available in group"):
-            assert group[0].fqdn == self.SECOND_HOST, f"Should be available host '{self.SECOND_HOST}'"
+@allure.step('Check error')
+def assert_that_host_add_is_unavailable(service_group: GroupConfig, host: Host):
+    with allure.step(f"Check that error is '{GROUP_CONFIG_HOST_ERROR.code}'"):
```

Comment:
Why code, not error?
Just a suggestion: maybe reverse quotes?

Reply:
I guess code will describe the problem, ex GROUP_CONFIG_HOST_ERROR

Reply:
Ah, okay, I've mistaken code for title